test/python/test_pack.py

```diff
@@ -224,6 +246,17 @@ def test_deb_pack(deb_archive, tmpdir):
             assert 'chown root:root /usr/lib/tmpfiles.d/{}.conf'.format(project.name) in postinst_script
 
 
+def test_rock_pack(rock_archive):
+    assert zipfile.is_zipfile(rock_archive.filepath), 'rock should be a valid zip-archive'
+    with zipfile.ZipFile(rock_archive.filepath, 'r') as zip_archive:
+        files = zip_archive.namelist()
+        # Tarantool 1.10 and Tarantool 2.2 use Luarocks 2 and Luarocks 3 respectively
+        # The content is this cases a bit different
+        # This checks is quite simple but shows that rock-archive is not empty
+        assert 'init.lua' in files, \
+            'Rock archive content is not as expected'
```

Comment:
Can you check that archive contains all `project.distribution_files`?

Reply:
This check will mean that we check that `tarantoolctl rocks pack` works correctly. I believe it's beyond responsibility of 'cartrige-cli'. I think such simple check should be enough

Reply:
The main idea of checking package contents is checking that some file was filtered out by `cartridge-cli` before running `tarantoolctl rocks pack`.

Reply:
No. In case of luarocks your logic doesn't work.

It depends on build type. You can specify list of needed files explicitly in your rockspec. You can choose "make" or "cmake" build type - and cmake and make will be responsible in this case.

Reply:
You can specify `build='none'` - and you will copy your sources

Reply:
But you definitely know that test project rockspec contains `source  = {{ url = 'file://.' }} build = {{ type = 'none' }}`.